staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go

```diff
@@ -384,6 +385,8 @@ func limitedReadBody(req *http.Request, limit int64) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
+	sizeObserved := float64(limit - lr.N)
```

Comment:
Isn't that `len(data)`? If it is then it's probably more obvious to the reader.

Reply:
Yeah, I'll remove this actually.